style.css

```diff
@@ -0,0 +1,525 @@
+body {
+  width: 375px;
```

Comment:
- [ ] [OPTIONAL] I notice that your webpage is not responsive 📱 because you set the width to 375px. It is advisable to have a design which looks good accross all devices 🖼️ In the project requierement, the **minimum width** is **375px** but **NOT** the only width. 😊 
To achieve that you can start by setting your width on the **body** element to 100%, and adjust all the width of the other elements accordingly ❤️